lib/sprinkler_web/live/temperature_live.ex

```diff
@@ -0,0 +1,18 @@
+defmodule SprinklerWeb.TemperatureLive do
+  use Phoenix.LiveView
+
+  @telemetry_topic "telemetry"
+
+  def mount(_params, _session, socket) do
+    Phoenix.PubSub.subscribe(Sprinkler.PubSub, @telemetry_topic)
+
+    {:ok, assign(socket, temp: "Unknown")}
+  end
+
+  def handle_info(
+        %{topic: @telemetry_topic, event: "new_reading", payload: %{temp: temp}},
```

Comment:
minor comment, but temp makes me feel we're talking about something "temporal" instead of temperature 😆 ignore this comment if we don't really care about this. I don't have any experience in Phoenix 😬

Reply:
Yeah I like `tmp` better but don’t mind that much